build/root/usr/local/bin/ci_entrypoint_gpu-operator.sh

```diff
@@ -4,11 +4,16 @@ set -o pipefail
 set -o errexit
 set -o nounset
 
+SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+
 prepare_cluster_for_gpu_operator() {
     trap collect_must_gather ERR
 
     toolbox/cluster/capture_environment.sh
-    entitle.sh
+
+    if [[ ${USE_ENTITLED_MIRROR} != "1" ]]; then
+        entitle.sh --mirror
+    fi
```

Comment:
not sure to understand this block, `if not use-the-entitled-mirror; then entitle.sh --mirror` ?

Reply:
Just old unpushed code, was focusing on pushing to `release-next`.... updated now